bedrock/firefox/templates/firefox/welcome/page1.html

```diff
@@ -0,0 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button with context %}
+
+{% add_lang_files "firefox/welcome/page1" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ _('More than a browser - Firefox products protect your life online') }}{% endblock %}
+{% block page_desc %}{{ _('Use Firefox Monitor to see if your info was compromised in another company’s data breach — and get automatically signed up for future alerts.') }}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+
+{% block body_class %}welcome-page1{% endblock %}
+
+{% block content_intro %}
+  <h1 class="c-page-title">{{ _('You’re on track to stay protected') }}</h1>
+  <p class="c-page-tagline">{{ _('You’ve got the web browser that protects your privacy — now it’s time to get a lookout for hackers.') }}</p>
+{% endblock %}
+
+{% block content_primary %}
+  <h2 class="body-primary-title"><img src="{{ static('img/logos/monitor/monitor-wordmark.svg') }}" width="256" height="" alt="Firefox Monitor"></h2>
+
+  <div class="body-primary-body">
+    {# L10n: "Firefox Monitor" is a product name and shouldn't be translated. #}
+    <p>{{ _('Firefox Monitor shows you if your information has been leaked in a known data breach, and alerts you in case it happens in the future.') }}</p>
+
+    <p class="primary-cta">{{ monitor_button(
+      button_id='monitor-button-signed-out',
```

Comment:
I don't think the monitor button JS is being initialized because you're passing a custom ID here. If you remove this line then ID defaults to `fxa-monitor-submit`, and the metrics request works ok.

Reply:
Oops, this was copypasta from WNP69.